ONSdigital/census-rm-case-api-2021
# Update entities

# Motivation and Context
Action Rules should exclude receipted Cases.

# What has changed
Entities updated in line with census-rm-case-processor
Integration test data generated using Easy Random

# How to test?
[trello](https://trello.com/c/GdA7flOq/788-rmc-152a-receipting-should-update-case-and-uac-active-flag-8)
[test as part of ](https://github.com/ONSdigital/census-rm-action-scheduler/pull/14)
